modulesets-stable/gtk-osx-python.modules

```diff
@@ -73,7 +73,9 @@
   <autotools id="python" autogenargs="--enable-shared"
 	     autogen-sh="configure" supports-non-srcdir-builds="no">
     <branch repo="python"
-	    module="2.7.13/Python-2.7.13.tar.xz" version="2.7.13">
+           module="2.7.14/Python-2.7.14.tar.xz" version="2.7.14"
+           hash="sha256:71ffb26e09e78650e424929b2b457b9c912ac216576e6bd9e7d204ed03296a66">
+      <patch file="https://git.gnome.org/browse/gtk-osx/plain/patches/python2-test_grammar.py-typo.patch" strip="1"/>
```

Comment:
Urq, did they have a regression on this? Is there a bug report upstream?

Reply:
Patch removed as it made no difference whatsoever:
https://github.com/totaam/gtk-osx-build/commit/c3645d4cde041db2f2500d4dfe53b171ce8b332f